infrastructure/terraform/modules/activation/variables.tf

```diff
@@ -34,3 +34,13 @@ variable "trigger_function_location" {
   type        = string
   default     = "us-central1"
 }
+
+variable "ga4_measurement_id" {
```

Comment:
See if it makes sense to add "sensitive   = true" to these variables. We should probably do it throughout the repo.